tests/jballerina-unit-test/src/test/resources/test-src/closures/closure.bal

```diff
@@ -751,3 +751,44 @@ function funcAcceptingIntArgs(int... scores) returns int {
     };
     return fn();
 }
+
+string moduleLevel = "module text";
+
+function errorConstructorReferenceTest1() returns error? {
```

Comment:
Since these functions aren't tests themselves shall we rename this and `errorConstructorReferenceTest2` to not include `Test`?